website/source/api/acl-tokens.html.md

```diff
@@ -268,6 +268,49 @@ $ curl \
 }
 ```
 
+## Read Self Token
+
+This endpoint reads the ACL token given by the passed SecretID. If the token is a global token
```

Comment:
Our terms are kind of confusing here. We say this returns an `ACL token` yet the HTTP header you pass is called `Token`. Then `SecretID` in this sentence refers to what you pass in the `Token` header.

I feel like I'm constantly in search of the term for what we use `aclObj` for in code: a token + its policies. I suppose that could be called an "ACL" or "ACL object"

Anyway, I might be the only one obsessed with terminology, and I don't have any good answers, so feel free to ignore. :)